flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/runtime/batch/sql/TimestampITCase.scala

```diff
@@ -41,12 +40,12 @@ class TimestampITCase extends BatchTestBase {
       Array(
         DataTypes.INT(),
         DataTypes.BIGINT(),
-        DataTypes.TIMESTAMP(9).bridgedTo(classOf[LocalDateTime]),
-        DataTypes.TIMESTAMP(9).bridgedTo(classOf[Timestamp]),
-        DataTypes.TIMESTAMP(3).bridgedTo(classOf[LocalDateTime]),
-        DataTypes.TIMESTAMP(3).bridgedTo(classOf[Timestamp]),
-        DataTypes.TIMESTAMP_WITH_LOCAL_TIME_ZONE(9).bridgedTo(classOf[Instant]),
-        DataTypes.TIMESTAMP_WITH_LOCAL_TIME_ZONE(9).bridgedTo(classOf[Instant]))
+        DataTypes.TIMESTAMP(9),
+        DataTypes.TIMESTAMP(9),
+        DataTypes.TIMESTAMP(3),
+        DataTypes.TIMESTAMP(3),
+        DataTypes.TIMESTAMP_WITH_LOCAL_TIME_ZONE(9),
+        DataTypes.TIMESTAMP_WITH_LOCAL_TIME_ZONE(9))
```

Comment:
Why change these?

Reply:
specifying conversion class of the types is not allowed in DDL and Descriptor API.